sklearn/ensemble/bagging.py

```diff
@@ -541,6 +541,23 @@ class BaggingClassifier(ClassifierMixin, BaseBagging):
         `oob_decision_function_` might contain NaN. This attribute exists
         only when ``oob_score`` is True.
 
+
+    Examples
+    --------
+    >>> from sklearn.svm import SVC
+    >>> from sklearn.ensemble import BaggingClassifier
+    >>> from sklearn.datasets import make_classification
+    >>> X, y = make_classification(n_samples=1000, n_features=4,
+    ...                            n_informative=2, n_redundant=0,
+    ...                            random_state=0, shuffle=False)
+    >>> clf = BaggingClassifier(n_estimators=100, random_state=0).fit(X, y)
+    >>> print(clf.predict([[0, 0, 0, 0]]))
```

Comment:
You don't need to `print`, you would see the output in the REPL anyway.

Reply:
I know, I just followed the suggestion from your last review. Should I revert this?

Reply:
I thought my suggestion was to remove it :P

![suggestion](https://user-images.githubusercontent.com/1663864/66210836-080f3d00-e6bb-11e9-91e4-48cccff2fbbb.png)

Reply:
Ha. I got confused, sorry about that.